samples/cellular/nrf_cloud_multi_service/overlay-nrf7002ek-wifi-scan-only.conf

```diff
@@ -50,3 +50,7 @@ CONFIG_NET_MAX_CONTEXTS=5
 
 # Disable LED patterns, enabling WiFi scanning takes control of two LEDs
 CONFIG_LED_INDICATION_DISABLED=y
+
+# Temporary tweak: Disable wifi_mgmt buffer offload; It causes scans to fail in scan-only mode.
+# To be removed when SHEL-3213 is resolved
+DCONFIG_NRF_WIFI_MGMT_BUFF_OFFLOAD=n
```

Comment:
@glarsennordic, spurious `D`

Reply:
Gah! I cannot believe I missed that. Thank you